app/assets/stylesheets/checkin.scss

```diff
@@ -1,9 +1,23 @@
 @import "colors";
 
 .disable-field-set {
-  background: $gray;
+  background: #ddd;
+  color: #ddd;
 }
 
 .disable-field-set h5 {
-  color: $shadow;
+  color: #eee;
+}
+
+.disable-field-set label {
+  color: #eee;
+}
+
+select:disabled {
+  color: #eee;
+}
+
+.disable-field-set input.submit {
+  background-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.